py3status/modules/battery_level.py

```diff
@@ -74,18 +81,21 @@
         (default None)
 
 Requires:
-    - the `acpi` command line
+    - the `acpi` the acpi command line utility (only if
+    `measurement_mode='acpi'`)
```

Comment:
This docstring should look like this to be parsed correctly (I realize that initially it was incorrect)
```
    acpi: command line utility (only if
        `measurement_mode='acpi'`)
```
If you could change this then I'll recommend this for merging.  Thanks for your hard work.